cc3d/core/PySteppables.py

```diff
@@ -105,6 +116,29 @@ def __init__(self, plot_name, plot_type, attribute_name, function_obj=None):
 
 
 class CellTypeFetcher:
+    """
+    Values assigned to cell types are accessible as attributes by name
+
+    For a CellType plugin specification as follows,
+
+    .. code-block:: xml
+
+        <Plugin Name="CellType">
+            <CellType TypeId="0" TypeName="Medium"/>
+            <CellType TypeId="1" TypeName="Condensing"/>
+            <CellType TypeId="2" TypeName="NonCondensing"/>
+        </Plugin>
+
+    Usage in Python can be performed as follows,
+
+    .. code-block:: python
+
+        cell_type_fetcher: CellTypeFetcher
+        x: int = cell_type_fetcher.CONDENSING  # = 1
```

Comment:
I forgot if we are capitalizing type names in cell_type_fetcher or not. Just something to double-check